Manatee.Trello/Rest/WebApiClient.cs

```diff
@@ -176,11 +177,15 @@ private static HttpContent GetContent(WebApiRestRequest request)
 					formData.Add(content, $"\"{parameter.Key}\"");
 				}
 
-				var byteContent = new ByteArrayContent(request.File);
-				formData.Add(byteContent, "\"file\"", $"\"{request.FileName}\"");
-				TrelloConfiguration.Log.Debug($"\tContent: {formData}");
 
-				return formData;
+                var fileStream = File.Open(request.File, FileMode.Open);
+                var fileInfo = new FileInfo(request.File);
+                formData.Add(new StreamContent(fileStream), "\"file\"", string.Format("\"{0}\"", "as" + fileInfo.Extension));
+                formData.Add(new StringContent("mimeType"), "image/png");
```

Comment:
I'd like to have some simple functionality that attempts to infer the mime type from the file extension.  Not looking for an extensive list; more like a "top 10."